requirements.in

```diff
@@ -7,4 +7,5 @@ omegaconf==2.1.0
 psycopg2-binary==2.9.1
 pyyaml==5.4.1
 requests==2.26.0
+requests-mock==1.9.3
```

Comment:
I don't think this should be in `requirements.in` if this is only used for testing. ATM this dependency will be packaged in every release, which increases size.

Instead, include this in `requirements-dev.in`.

Reply:
Done.